apps/main/src/lend/entities/chain/chain-query.ts

```diff
@@ -6,9 +6,11 @@ import { ChainId } from '@/lend/types/lend.types'
 
 export const { useQuery: useOneWayMarketNames, prefetchQuery: prefetchMarkets } = queryFactory({
   queryKey: ({ chainId }: ChainParams) => ['chain', { chainId }, 'markets'] as const,
-  queryFn: async (_: ChainQuery<ChainId>): Promise<string[]> => {
+  queryFn: async (chainId: ChainQuery<ChainId>): Promise<string[]> => {
+    const useAPI = chainId.chainId !== 146 // disable API for sonic
```

Comment:
```suggestion
    const useAPI = true
```

Reply:
Turns out this cannot yet be done, as gauges don't seem to work yet for the API